s3-connector/unittest_s3-connector.py

```diff
@@ -33,20 +34,18 @@ def test_connection(self):
         self.assertTrue(self.s3_connection)
```

Comment:
As discussed, this test will always pass as it's not testing the connection! So I would recommend deleting it.

Reply:
@orionsBeltWest As per your recommendation I have added both functions downlaod/upload objects. Also, I have removed the test connection.